zng/type.go

```diff
@@ -216,6 +216,15 @@ func ContainedType(typ Type) (Type, []Column) {
 	}
 }
 
+func IsUnionType(typ Type) bool {
+	switch typ.(type) {
+	case *TypeUnion:
+		return true
+	default:
+		return false
+	}
+}
```

Comment:
I don't think this is worth adding since folks can just say `_, ok := typ.(*zng.TypeUnion)`, but if you do want it, please shorten to
```go
_, ok := typ.(*zng.TypeUnion)
return ok
```

Reply:
This is here because `_, ok := type.(foo)` doesn't compose in boolean expressions. Shortening per your suggestion.